api/src/modules/scenarios/cost-surface/__mocks__/cost-surface-events-fake.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable, Scope } from '@nestjs/common';
+import {
+  CostSurfaceEventsPort,
+  CostSurfaceState,
+} from '../cost-surface-events.port';
+
+@Injectable()
```

Comment:
Little clarifications. The reason to making Mock services @Injectable is no get them as provider when you create sandboxes or SUT's?

Reply:
@alexeh yes - to let them be initialized by NestJS DI.